### ISSUE TITLE
[BUG] [Service]Need to request several times to get translation with Intento MT.

### ISSUE THREAD

**Issue by lyiyu66** [vmware/singleton#989]
**Describe the bug**
MT to Intento doesn't work. 

**To Reproduce**
1. Set the below data to application_bundle.properties:
```
mt.server=https://api.inten.to/ai/text/translate
mt.key=
mt.intento.apikey=
mt.intento.category=d30ccd9e-7e11-471c-9aa6-8b5983115f2d-VMwareFromEN-TECH
mt.intento.provider=ai.text.translate.microsoft.translator_text_api.3-0
mt.intento.transURL=https://api.inten.to/operations
```
2. Go to  'https://localhost:8090/i18n/api/v2/translation/products/vmware2018/versions/1.1.0/locales/en/components/custom?checkTranslationStatus=false&machineTranslation=true&pseudo=false
3. See response:
```
{
  "response": {
    "code": 404,
    "message": "Request failed: respond empty from MT server!",
    "serverTime": ""
  },
  "signature": "",
  "data": null
}
```

**Expected behavior**
Can get translation from Intento server.

**Comment by lyiyu66** [comment]
After all settings in application file are needed, Intento MT server can be connected. 
But always need to request several times for getting the translation. Hence edit the bug title...

**Comment by lyiyu66** [comment]
The issue is fixed with commit 9076cfb62d38f18df9bc300baf294d957c244901
